.github/workflows/push.yml

```diff
@@ -43,6 +52,8 @@ jobs:
           - 27017:27017
     env:
       MONGO_DB_URL: mongodb://localhost:27017/talawa-test-db
+      REDIS_HOST: localhost
```

Comment:
why is these value hard-coded?

Reply:
Ohhh no, that is not the config file that is the workflow file in CI/CD. Basically to test the application in the CI/CD, we have to provide the env variables for connection to a MongoDb database and redis instance. Otherwise the server will crash in the container.

I started the redis service inside the testing container and passed the env variables to connect to those services.

They need to be hard-coded because we have to pass the actual value for the connection to happen in the application within the container.